network/src/network.rs

```diff
@@ -76,13 +76,14 @@ pub struct SessionInfo {
 pub struct NetworkState {
     pub(crate) peer_registry: RwLock<PeerRegistry>,
     pub(crate) peer_store: Mutex<Box<dyn PeerStore>>,
-    pub(crate) original_listened_addresses: RwLock<Vec<Multiaddr>>,
-    dialing_addrs: RwLock<FnvHashMap<PeerId, Instant>>,
+    pub(crate) original_listened_addrs: RwLock<Vec<Multiaddr>>,
```

Comment:
It wastes reviewers' time to mess up a PR with renaming refactoring.

Reply:
Just `addresses` to `addrs`